homeassistant/components/fujitsu_hvac/climate.py

```diff
@@ -0,0 +1,188 @@
+"""Support for Fujitsu HVAC devices that use the Ayla Iot platform."""
+from typing import Any
+
+from ayla_iot_unofficial.fujitsu_hvac import Capability, FujitsuHVAC
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, PRECISION_HALVES, UnitOfTemperature
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    DOMAIN,
+    FUJI_TO_HA_FAN,
+    FUJI_TO_HA_HVAC,
+    FUJI_TO_HA_SWING,
+    HA_TO_FUJI_FAN,
+    HA_TO_FUJI_HVAC,
+    HA_TO_FUJI_SWING,
+)
+from .coordinator import FujitsuHVACCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up one Fujitsu HVAC device."""
+    api = hass.data[DOMAIN][entry.entry_id]
+    coordinator = FujitsuHVACCoordinator(hass, api)
+    await coordinator.async_config_entry_first_refresh()
+
+    async_add_entities(
+        [FujitsuHVACDevice(coordinator, dev) for dev in coordinator.data.values()]
+    )
+
+
+class FujitsuHVACDevice(CoordinatorEntity, ClimateEntity):
```

Comment:
```suggestion
class FujitsuHVACDevice(CoordinatorEntity[..], ClimateEntity):
```
Please add the coordinator in the type